### PR TITLE
1.3 notes update

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: _includes/manuals/1.2/3.5.5_importing_facts.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you are planning using Foreman as your Puppet [External Nodes](#) Classifier,
 
 If you run Foreman on the same machine as your puppetmaster and you have only one puppet master, you probably want to create a simple cronjob:
 <pre>
-rake puppet:import:hosts_and_facts RAILS_ENV=production
+foreman-rake puppet:import:hosts_and_facts
 </pre>
```
Comment (GregSutcliffe) [inline _includes/manuals/1.2/3.5.5_importing_facts.md:12]: foreman-rake was only added in 1.3, iirc?

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 2]
```diff
@@ -5,13 +5,15 @@
...
-installation section of the manual to see how it works.
+A new installer based on the
+[kafo](https://github.com/theforeman/kafo) project is available. Refer to the
+[installation section](/manuals/1.3/index.html#3.2ForemanInstaller) of the
+manual for more information.
 
 #### Smart-proxy auto-registration
 
-In the default setup, smart-proxy is automatically registered using Foreman API
+In the default all-in-one setup, the smart-proxy is automatically registered
```
Comment (GregSutcliffe) [inline _includes/manuals/1.3/1.2_release_notes.md:15]: nitpick: "automatically registered in the Foreman UI" may be the clearest statement

File: _includes/manuals/1.3/1.2_release_notes.md [hunk 3]
```diff
@@ -139,8 +149,8 @@ removal of JSON roots ([#2803](http://projects.theforeman.org/issues/2803)).
 #### New foreman-rake command
 
 There is new command `foreman-rake` which must be used instead `rake` for doing
-ADMINISTRative tasks. It is a wrapper around `rake` that changes to the correct
-directory and on SCL-enabled systems it does use `scl` command.
+administrative tasks. It is a wrapper around `rake` that changes to the correct
+directory and on SCL-enabled systems it uses the `scl` command.
```
Comment (GregSutcliffe) [inline _includes/manuals/1.3/1.2_release_notes.md]: is this not a duplicate of line 77 above?

File: _includes/manuals/1.3/3.5.2_configuration_options.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ Default: ["fqdn", "hostgroup", "os", "domain"]
 
 ##### document_root
 
-This is the directory into which the *rake puppet:rdoc:generate* task will place the web-based on-line documentation for your puppet classes. It should be under a web server's control and the default location is served by Foreman's own web server.
+This is the directory into which the *foreman-rake puppet:rdoc:generate* task will place the web-based on-line documentation for your puppet classes. It should be under a web server's control and the default location is served by Foreman's own web server.
 Default: public/puppet/rdoc
```
Comment (GregSutcliffe) [inline _includes/manuals/1.3/3.5.2_configuration_options.md]: i'm not sure if the rdoc task works at all at the moment, do we want to keep this?

**Comment by GregSutcliffe** [comment]
@domcleal a few small points, looks good overall

File: _includes/manuals/1.2/3.5.5_importing_facts.md (hunk 1 above)
Comment (domcleal) [inline _includes/manuals/1.2/3.5.5_importing_facts.md:12]: Good catch, git rebase appears to have helpfully updated the 1.2 manual instead after your PR was merged!

File: _includes/manuals/1.3/1.2_release_notes.md (hunk 3 above)
Comment (domcleal) [inline _includes/manuals/1.3/1.2_release_notes.md]: Yes, it's in both headline features and upgrade notes, not sure which it should stay in really.  Maybe just upgrade notes?

File: _includes/manuals/1.3/3.5.2_configuration_options.md (hunk 4 above)
Comment (domcleal) [inline _includes/manuals/1.3/3.5.2_configuration_options.md]: Yeah, it's broken.  Just remove the setting from the docs altogether?

Comment (GregSutcliffe) [inline _includes/manuals/1.3/3.5.2_configuration_options.md]: Wfm

File: _includes/manuals/1.3/1.2_release_notes.md (hunk 3 above)
Comment (GregSutcliffe) [inline _includes/manuals/1.3/1.2_release_notes.md]: Yeah, i don't think it's a headline, it was only added for _our_ quality of life really :P

**Comment by domcleal** [comment]
@GregSutcliffe thanks, updated!

**Comment by GregSutcliffe** [comment]
:+1:

**Comment by lzap** [comment]
Thanks!

**Pull request closed without merging**